guides/doc-Provisioning_Hosts/master.adoc

```diff
@@ -4,6 +4,12 @@ include::common/header.adoc[]
 
 = {ProvisioningDocTitle}
 
+// This guide is not ready for stable releases
```

Comment:
same as for "managing hosts" above: I believe we can/should hide smaller parts; more specifically those that mention Katello-only features.

Reply:
My idea is that we should certainly do that, but this provides a good base for it.

For example, https://docs.theforeman.org/nightly/Provisioning_Hosts/index-foreman-deb.html#using-a-synced-kickstart-repository_provisioning is a section we should hide. https://docs.theforeman.org/nightly/Provisioning_Hosts/index-foreman-deb.html#Creating_an_Installation_Medium_for_Debian-Ubuntu_provisioning contains an unreplaced macro.

Again, doing everything in a single PR is massive. What I'd like to do is gather a list of guides that are ready and start exposing those in the navigation. Then one by one we can review guides and expose them.